### ISSUE TITLE
risk ban?

### ISSUE THREAD

**Issue by Suburbanno** [Aelto/Stava#1]

**Comment by Aelto** [comment]
**You don't risk anything by using Stava**, all it does is using and grouping multiple public services offered by RiotGames themselves  (API endpoints)  into what makes the final app.
It doesn't give you any advantage over someone not using Stava as it _just_ retrieves informations on all players (information you could get on [op.gg](http://op.gg/) for example) in your games and pack them into a single screen.
It doesn't touch anything inside your league of legends folder neither.
**In short, it doesn't break anything in the LoL terms of use.**

The next step would be that i register the app on the [developer.riotgames](https://developer.riotgames.com/) website, but it isn't the subjet of this issue so i'm closing it.
